### PR TITLE
feat: Import multiple image bundles

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: cmd/importcmd/imagebundle/image_bundle.go [hunk 1]
```diff
@@ -43,22 +44,42 @@ func NewCommand(out output.Output) *cobra.Command {
...
-			out.StartOperation("Parsing image bundle config")
-			cfg, err := config.ParseImagesConfigFile(filepath.Join(tempDir, "images.yaml"))
-			if err != nil {
-				out.EndOperation(false)
-				return err
+			var cfg config.ImagesConfig
+
+			// Just in case users specify the same bundle twice, keep a track of
+			// files that have been extracted already so we only extract each of them once.
+			extractedBundles := make(map[string]struct{}, len(imageBundleFiles))
```
Comment (aweris) [inline cmd/importcmd/imagebundle/image_bundle.go:53]: [nit] I think with type bool, intend would be easier to understand.

```suggestion
			extractedBundles := make(map[string]bool, len(imageBundleFiles))
```

Comment (dkoshkin) [reply cmd/importcmd/imagebundle/image_bundle.go:53]: @jimmidyson already used this format in a different PR, I would prefer consistency.

File: cmd/importcmd/imagebundle/image_bundle.go [hunk 2]
```diff
@@ -43,22 +44,42 @@ func NewCommand(out output.Output) *cobra.Command {
...
+				err = archive.UnarchiveToDirectory(imageBundleFile, tempDir)
+				if err != nil {
+					out.EndOperation(false)
+					return fmt.Errorf("failed to unarchive image bundle: %w", err)
+				}
+				out.EndOperation(true)
+
+				out.StartOperation("Parsing image bundle config")
+				bundleCfg, err := config.ParseImagesConfigFile(
+					filepath.Join(tempDir, "images.yaml"),
```
Comment (dkoshkin) [inline cmd/importcmd/imagebundle/image_bundle.go:70]: Took a little to understand how this works, but this `images.yaml` will always be overwritten and from the bundle in this loop, and then merged with the rest of the cfg.

**Review by dkoshkin — APPROVED** [review]

File: cmd/importcmd/imagebundle/image_bundle.go (hunk 1 above)
Comment (jimmidyson) [reply cmd/importcmd/imagebundle/image_bundle.go:53]: It's pretty common practice to use `struct{}` as marker in map.

File: cmd/importcmd/imagebundle/image_bundle.go (hunk 2 above)
Comment (jimmidyson) [reply cmd/importcmd/imagebundle/image_bundle.go:70]: Ah yeah let me add a comment

Comment (jimmidyson) [reply cmd/importcmd/imagebundle/image_bundle.go:73]: Added in 73e4c43003159de58b1f3b799fa76b31666510f8

**Pull request merged**